test/plugin/test_exception_detector.rb

```diff
@@ -270,6 +270,48 @@ class ExceptionDetectorTest < Test::Unit::TestCase
   puma (3.10.0) lib/puma/thread_pool.rb:120:in `block in spawn_thread'
 END
 
+  DART_EXC = <<END.freeze
+Unhandled exception:
+some uncaught exception
+#0      handleFailure (file:///test/example/http/handling_an_httprequest_error.dart:16:3)
+#1      main (file:///test/example/http/handling_an_httprequest_error.dart:24:5)
+<asynchronous suspension>
+#2      _startIsolate.<anonymous closure> (dart:isolate-patch/isolate_patch.dart:265)
+#3      _RawReceivePortImpl._handleMessage (dart:isolate-patch/isolate_patch.dart:151)
+END
+
+  DART_COMPLEX_EXC = <<END.freeze
```

Comment:
`COMPLEX_DART_EXC`, for consistency with `COMPLEX_JAVA_EXC`.

Reply:
Updated!